master/static/migrations/20240520104140_add-run-hparam-table.tx.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE run_hparams (
```

Comment:
we drop ```projects.hyperparameters``` and delete the associated code now that it is replaced by this?

Can do this in a follow up ticket if you want